FlowCrypt/Controllers/Setup/SetupInitialViewController.swift

```diff
@@ -53,19 +53,22 @@ final class SetupInitialViewController: TableNodeViewController {
     private let user: UserId
     private let router: GlobalRouterType
     private let decorator: SetupViewDecorator
+    private let organisationalRules: OrganisationalRules?
 
     private lazy var logger = Logger.nested(in: Self.self, with: .setup)
 
     init(
         user: UserId,
         backupService: BackupServiceType = BackupService(),
         router: GlobalRouterType = GlobalRouter(),
-        decorator: SetupViewDecorator = SetupViewDecorator()
+        decorator: SetupViewDecorator = SetupViewDecorator(),
+        organisationalRulesService: OrganisationalRulesServiceType = OrganisationalRulesService()
     ) {
         self.user = user
         self.backupService = backupService
         self.router = router
         self.decorator = decorator
+        self.organisationalRules = organisationalRulesService.getSavedOrganisationalRulesForCurrentUser()
```

Comment:
Same question here - can it be required?

Reply:
done